cli/utils.py

```diff
@@ -0,0 +1,13 @@
+RG_NAME_PATTERN = '^[-\w\._\(\)]{1,90}[^\.]$'
+LOCATION_PATTERN = '^[a-zA-Z]+2?$'
+
+def already_exists(resource_type, resource_name):
+    return '{} {} already exists.'.format(resource_type, resource_name)
+
+def created(resource_type, resource_name):
+    return 'Created {} {}.'.format(resource_type, resource_name)
+
+def create_failed(resource_type, resource_name, error_message):
+    return 'Failed to create {} {}. Exception:{}'.format(resource_type,
+                                                         resource_name,
+                                                         error_message)
```

Comment:
again, this long lines with short danglers thing looks odd, but if its the python way of doing it...

Reply:
In this case since all params fit within the 80 chars I think its best to go with this way as its a more pythonic way (the other case above had really long lines after 80 if I did this)